packages/terra-time-input/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
```

Comment:
I'm thinking we may want to treat this as a major version bump. Thoughts cc/ @ryanthemanuel @mjhenkes @tbiethman @neilpfeiffer

Reply:
I agree since we're getting rid of the meridiem select and this would be a breaking change.

Reply:
If this is a MVB, then I shall add the upgrade guide soon after the CSS changes are validated.

Reply:
After further discussions with the Terra team, it was decided that the changes on this pull request will not be considered a breaking change because there are no changes to the API. Therefore, this does not require a major version bump. Since there are layout and visual changes to the meridiem buttons, we expect some WDIO tests to fail for consumers and those screenshots would need to be regenerated. In rare cases where consumer may have logic in their application to query for the old meridiem select or listen to any native events, that logic would need to be updated.

@nramamurth An upgrade guide would not be needed but it would be good to add additional documentation in both the terra-date-time-picker and terra-time-input changelogs to call out that the meridiem select normally shown on desktop is now changed to buttons. It would also be helpful for consumers by documenting on the changelogs that WDIO screenshots and any code in the consumer application that accesses the old meridiem select may need to be updated.

Reply:
@benbcai Gotcha! I have [updated the Changelog](https://github.com/cerner/terra-framework/pull/837/commits/7dcf2192f7452dd9ec6905c0ce2ecfd2fe76d443). Lmk if this is not sufficient.